SPECS/glide/glide.spec

```diff
@@ -0,0 +1,87 @@
+Summary:        Vendor Package Management for Goland
```

Comment:
"golang" instead of "goland"?

Reply:
This is inherited from PhotonOS and shows as Goland. Doesn't look like a typo. 
https://github.com/vmware/photon/blob/master/SPECS/glide/glide.spec

Reply:
Goland is clearly a typo and we should fix it. It says "Glide: Vendor Package Management for Golang" right at the top of the README